compiler/cpp/src/thrift/generate/t_py_generator.cc

```diff
@@ -127,6 +128,9 @@ class t_py_generator : public t_generator {
         gen_tornado_ = true;
       } else if( iter->first.compare("coding") == 0) {
         coding_ = iter->second;
+      } else if( iter->first.compare("type_hints") == 0) {
+        gen_type_hints_ = true;
+        gen_enum_ = true;
```

Comment:
Would it be worth doing:

```
    if (!gen_enum) { throw "the type_hints py option requires the enum py option}
```
instead of setting it to true?

Reply:
I agree. Updated.